### PR TITLE
API overview - <Circle /> and <Ellipse /> components

### PR DESCRIPTION
**Pull request by christo-pr**
### This PR will

* Create a `<Circle />` component
* Create a `<Ellipse />` component

**Note:**
 Still WIP since we are going to resuse the `<Circle />` component to render the **arc** and the **ellipse** components.

LegraJS Doc:
> Draws an ellipse/circle with the center at (xc, yc). For the ellipse, a and b are the horizontal and vertical semi-axis lengths respectively.

> An arc is described as a **section** of en ellipse. `x`, `y` represent the center of that ellipse. `a` and `b` are the horizontal and vertical semi-axis lengths, respectively, for that ellipse.
`start`, `stop` are the start and stop angles for the arc.
`closed` is a boolean argument. If true, lines are drawn to connect the two end points of the arc to the center.

### CODE REVIEW

File: src/components/Circle.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    if (cnvs) {
+      const ctx = cnvs.getContext('2d');
+      const legra = new Legra(ctx, bs);
+
+      // Reset canvas only on basic render
+      if (!ctx) {
+        ctx.clearRect(0, 0, cnvs.width, cnvs.height);
+      }
+
+      // Draw the linear path
```
Comment (fernando-barajas) [inline src/components/Circle.js]: @christo-pr copy/paste??

File: src/components/Ellipse.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    if (cnvs) {
+      const ctx = cnvs.getContext('2d');
+      const legra = new Legra(ctx, bs);
+
+      // Reset canvas only on basic render
+      if (!ctx) {
+        ctx.clearRect(0, 0, cnvs.width, cnvs.height);
+      }
+
+      // Draw the linear path
```
Comment (fernando-barajas) [inline src/components/Ellipse.js]: @christo-pr same here

File: src/components/Circle.js (hunk 1 above)
Comment (christo-pr) [reply src/components/Circle.js]: xD

File: src/components/Ellipse.js (hunk 2 above)
Comment (christo-pr) [reply src/components/Ellipse.js]: :shame:

**Pull request merged**